.eslintrc.js

```diff
@@ -1,25 +1,29 @@
+require('@rushstack/eslint-patch/modern-module-resolution')
+
 module.exports = {
   env: {
     browser: true,
     es2021: true,
     node: true,
   },
-  parser: 'vue-eslint-parser',
-  extends: ['eslint:recommended', 'plugin:vue/essential', 'plugin:@typescript-eslint/recommended'],
+  extends: [
+    'eslint:recommended',
+    'standard',
+    'plugin:vue/vue3-recommended',
```

Comment:
main change based on the discussion https://github.com/themesberg/flowbite-vue/discussions/200
it introduces stricter rules based on official `eslint-plugin-vue`